src/ui-event.c

```diff
@@ -351,7 +351,7 @@ void keypress_to_readable(char *buf, size_t len, struct keypress src)
  */
 bool char_matches_key(wchar_t c, keycode_t key)
 {
-	wchar_t keychar[2];
+	wchar_t keychar[2] = {L'\0', L'\0'};
```

Comment:
I would drop the added initialization and replace the last two lines of the function ("text_mbstowcs(keychar, k, 1); return (c == keychar[0]);") with "return text_mbstowcs(keychar, k, 1) > 0 && c == keychar[0];".